pom.xml

```diff
@@ -12,13 +12,12 @@
     <description>Offers matrix-based security authorization strategies (global and per-project).</description>
     <url>http://wiki.jenkins-ci.org/display/JENKINS/Matrix+Authorization+Strategy+Plugin</url>
     <properties>
-      <findbugs-maven-plugin.version>3.0.1</findbugs-maven-plugin.version>
-      <findbugs.failOnError>true</findbugs.failOnError>
+      <jenkins.version>1.625.3</jenkins.version> <!-- must have a65d007 -->
```

Comment:
https://github.com/jenkinsci/jenkins/commit/a65d007 by @Vlatombe. Not needed for _running_ the plugin, but needed for _testing_ it: with the `cloudbees-folder` dep, we must also have the `credentials` dep, and then you get a plugin cycle without this core patch. Cf. [JENKINS-28942](https://issues.jenkins-ci.org/browse/JENKINS-28942).